docs/source/models/gbd2017/causes/ischemic_stroke/index.rst

```diff
@@ -48,15 +80,150 @@ Model Assumptions and Limitations
 
 Cause Model Diagram
 -------------------
+
+According to GBD 2017, stroke cases are considered acute from the day of incidence of a first-ever stroke through day 28 following the event. Chronic stroke includes the sequelae of an acute stroke AND all recurrent stroke events. Stroke cases are considered chronic beginning 28 days following the occurrence of an event. Chronic stroke includes the sequelae of an acute stroke AND all recurrent stroke events.
+
 .. image:: ischemic_stroke_transitions.svg
 
 
 Data Description
 ----------------
 
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
+.. list-table:: State Definitions
+   :widths: 1, 5, 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - A
+     - **A**\ cute Ischemic Stroke
+     - Simulant that is in duration-based period starting day of incidence of a first-ever stroke through day 28 following the event
+   * - C
```

Comment:
Consider changing *C*hroinc to *P*ost for stroke, analogous to what we discussed for IHD yesterday.  Also include susceptible in this list